tools/stress-cluster/cluster/kubernetes/stress-test-addons/values.yaml

```diff
@@ -34,8 +38,9 @@ subscription:
 tenantId:
   test: 72f988bf-86f1-41af-91ab-2d7cd011db47
   prod: 72f988bf-86f1-41af-91ab-2d7cd011db47
-  dev: 72f988bf-86f1-41af-91ab-2d7cd011db47
```

Comment:
Any reason for deleting this?

Reply:
I thought it would just be more consistent with dev fields being empty. And tenant could be different after all if you were running these scripts outside the microsoft corp tenant.